src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1852,13 +1837,71 @@
     </Field>
     <Field Name="Value" Type="ExpressionSyntax"/>
   </Node>
+
+
+  <AbstractNode Name="MultiVariableDeclarationSyntax" Base="CSharpSyntaxNode">
```

Comment:
It is not a base of `VariableDeclarationSyntax`.

---
In reply to: [71962344](https://github.com/dotnet/roslyn/pull/12688#discussion_r71962344) [](ancestors = 71962344)